docs/common.py

```diff
@@ -80,6 +84,12 @@
 
 .. |release_tarball_kafka10| replace:: %(url_locationtech_release)s/geomesa-kafka-10-dist_2.11/%(release)s/geomesa-kafka-10-dist_2.11-%(release)s-bin.tar.gz
 
+.. |release_tarball_hbase| replace:: %(url_locationtech_release)s/geomesa-hbase-dist_2.11/%(release)s/geomesa-hbase-dist_2.11-%(release)s-bin.tar.gz
+
+.. |release_tarball_bigtable| replace:: %(url_locationtech_release)s/geomesa-bigtable-dist_2.11/%(release)s/geomesa-bigtable-dist_2.11-%(release)s-bin.tar.gz
```

Comment:
this might not be used anywhere so it might not matter - but we don't release the bigtable dist on locationtech